powertools-sqs/src/main/java/software/amazon/payloadoffloading/PayloadS3Pointer.java

```diff
@@ -0,0 +1,59 @@
+package software.amazon.payloadoffloading;
```

Comment:
Is this a new risk? 

If the offloading library changes it's implementation of PayloadS3Pointer what do we do?

Reply:
Its a acceptable risk at this point. I dont believe though its possible for payload lib to change the implementation and keep it backward compatible. That being said, In future version, we would like to move away from hard coded dependency of offloaded msg format and make it more customizable for users to pass it to us. Ref https://github.com/awslabs/aws-lambda-powertools-java/issues/603

Reply:
👍